src/routes/Dashboard/components/Dashboard.scss

```diff
@@ -5,9 +5,18 @@
 .tabs {
   width: 100%;
   height: 600px;
+  overflow: scroll;
 }
 
 .map {
   width: 100%;
   height: 600px;
 }
+
+pre {
```

Comment:
I kind of like it. It gives the demoer the ability to show the json data and see any info not rendered in the map. But a URL to the API would probably also suffice